sleap/gui/app.py

```diff
@@ -1120,6 +1148,35 @@ def _update_gui_state(self):
 
         # Update menus
 
+        def _update_sessions_menu(self, frame_idx):
+            """Update the instance groups menu based on the selected frame index in the session."""
+            session = self.state.get("session")
```

Comment:
You can just use:
```suggestion
            session = self.state["session"]
```
which follows convention (and makes our code easier to search)